Client lib used by server code is missing from the 2.6.0-rc.1 release

[This file](https://github.com/thelounge/lounge/blob/master/src/plugins/irc-events/link.js#L7) is trying to load another file from [`client/js/...`](https://github.com/thelounge/lounge/blob/master/client/js/libs/handlebars/ircmessageparser/findLinks.js), which are [ignored in our release](https://github.com/thelounge/lounge/blob/master/.npmignore#L9) since [#1611](https://github.com/thelounge/lounge/pull/1611/files#diff-774bc7673169ae00d4c2bd961ab79828R4), as shown [here](https://unpkg.com/thelounge@2.6.0-rc.1/):

```
(module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
2017-11-12 05:21:18 [INFO] The Lounge v2.6.0-rc.1 (node 4.8.6 on linux x64)
2017-11-12 05:21:18 [INFO] Configuration file: /etc/lounge/config.js
2017-11-12 05:21:18 [INFO] Available at http://127.0.0.1:9000/ in private mode
module.js:327
    throw err;
    ^

Error: Cannot find module '../../../client/js/libs/handlebars/ircmessageparser/findLinks'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/lib/node_modules/thelounge/src/plugins/irc-events/link.js:7:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

There was a recent issue with `webpack-cleanup-plugin` deleting all files in `client/` I believe (see https://github.com/thelounge/lounge/pull/1682). Could it be that `webpack-cleanup-plugin` cleans up the same things than listed in the .npmignore file?

--- a/.npmignore
+++ b/.npmignore
@@ -6,7 +6,11 @@
 .*
 !.lounge_home
 
+# Ignore client folder as it's being built into public/ folder
+# except for the findLinks.js file which is used by the server
 client/
+!client/js/libs/handlebars/ircmessageparser/findLinks.js
+
 public/js/bundle.vendor.js.map
 coverage/
 scripts/

